src/routes/raw.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import APIRouter
+from src.data.api import handler
+
+api = handler.api
+
+router = APIRouter()
+
+
+@router.get("/raw/all", status_code=200)
+async def get_all_raw(source: str = ""):
+    """
+    All AHGS API Calls Only
+    """
+    update = await api.update_all()
```

Comment:
is there caching?
Unless I'm missing something this would make a call to AH everytime?

Reply:
Caching/time delay happens in src/data/api/api.py, handled by the update_all() function

Reply:
Looks like it wasn't handling it correctly, fix pushed bleow